### PR TITLE
EI-182 - zendesk

### PR DESCRIPTION
**Pull request by gdbarnes**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Web/Infrastructure/Configuration/WebConfigurationOptions.cs [hunk 1]
```diff
@@ -3,11 +3,15 @@
     public class WebConfigurationOptions
     {
         public const string EmployerIncentivesWebConfiguration = "EmployerIncentivesWeb";
-
         public virtual string RedisCacheConnectionString { get; set; }
         public virtual string CommitmentsBaseUrl { get; set; }
         public virtual string AccountsBaseUrl { get; set; }
         public virtual string AllowedHashstringCharacters { get; set; }
         public virtual string Hashstring { get; set; }
+        public virtual string CdnBaseUrl { get; set; }
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Web/Infrastructure/Configuration/WebConfigurationOptions.cs:11]: CdnBaseUrl  and ZenDeskHelpCentreUrl  aren't being used.

File: src/SFA.DAS.EmployerIncentives.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -24,6 +24,8 @@
         <!-- End Google Tag Manager -->
     </environment>
 
+    @Html.Partial("_ZenDeskWidget")
```
[the remark this answers is not in the document]
Comment (gdbarnes) [reply src/SFA.DAS.EmployerIncentives.Web/Views/Shared/_Layout.cshtml:27]: @Stephek Did you mean to put a code example in there?

File: src/SFA.DAS.EmployerIncentives.Web/Infrastructure/Configuration/WebConfigurationOptions.cs (hunk 1 above)
Comment (gdbarnes) [reply src/SFA.DAS.EmployerIncentives.Web/Infrastructure/Configuration/WebConfigurationOptions.cs:11]: No worries. I'll take them out for now.

File: src/SFA.DAS.EmployerIncentives.Web/Views/Shared/_Layout.cshtml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gdbarnes) [reply src/SFA.DAS.EmployerIncentives.Web/Views/Shared/_Layout.cshtml:27]: This maybe?
`@await Html.PartialAsync("_ZenDeskWidget")`

[the remark this answers is not in the document]
Comment (gdbarnes) [reply src/SFA.DAS.EmployerIncentives.Web/Views/Shared/_Layout.cshtml]: Changed import to `<partial name="_PartialName" />` as it renders async.

**Pull request merged**